developer-docs-site/docs/nodes/full-node/fullnode-source-code-or-docker.md

```diff
@@ -158,6 +158,12 @@ curl -O https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/mainnet
 curl -O https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/mainnet/genesis.blob
 ```
 
+TODO: Find out if this `mainnet` directory should reside in the `aptos-core` checkout as when attempting to start the fullnode indexer with the directory distinct results in the error:
+```
+➜  mainnet cargo run --bin aptos-node --features "indexer"  -- --config fullnode.yaml
```

Comment:
cargo run -p aptos-node --features "indexer" --release -- -f /path/to/fullnode.yaml